lib/instrumentation/transaction.js

```diff
@@ -127,9 +127,7 @@ Transaction.prototype._encode = function (cb) {
       duration: self.duration(),
       timestamp: new Date(self._timer.start).toISOString(),
       result: String(self.result),
-      sampled: self.sampled,
-      context: null,
-      spans: null
```

Comment:
@Qard I hope it's ok that I removed the two `null` properties here? As far as I remember you added these at some point. Is there a performance or other reason for them being here?

Reply:
Yeah, I put them there to avoid the [hidden class](https://gist.github.com/twokul/9501770) problem.

Reply:
I thought that didn't have any effect on anonymous objects like this one? If it does, then there's a few places we should add this as well 😃

Reply:
It impacts any kind of object. It gets allocated with the initial shape and then any future additions will create a new hidden class linking back to it.

Reply:
Didn't know that. I wonder how it can do that. I think I'll do a little research into this. For now I've restored it to look like before